tests/test_utils.py

```diff
@@ -187,6 +187,7 @@ class TestParseGHRepo:
         'github.com/foo/bar.git',
         'www.github.com/foo/bar',
         'http://github.com/foo/bar',
+        'https://github.com/foo/bar/bar'
```

Comment:
What about `https://github.com/foo/bar/bar/anotherbar`? :P

Reply:
@msrb I guess now it's good to handle any such cases. It will only take into consideration the first two elements which is `org+reponame`